ports/servoshell/prefs.rs

```diff
@@ -374,6 +373,13 @@ pub(crate) fn parse_command_line_arguments(args: Vec<String>) -> ArgumentParsing
         "--prefs-file /path/to/prefs.json",
     );
 
+    opts.optopt(
+        "",
+        "resource-dir",
```

Comment:
I would prefer not to have specialized behavior for ohos. My idea would be that a potentially bundled `prefs.json` (in the resource dir) could be copied into the config dir, if it doesn't exist there. 
We will eventually allow prefs to be changed at runtime, and then we probably also want to persist the state back to the file in the config directory.